client/src/components/DatasetDownloadFileSummary.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Box, Text } from 'grommet'
+import { config } from 'config'
+import { getReadable } from 'helpers/getReadable'
+import { DatasetSummaryTable } from 'components/DatasetSummaryTable'
+import { Link } from 'components/Link'
+
+export const DatasetDownloadFileSummary = ({ dataset }) => {
+  const filesSummary = dataset.stats.files_summary
+
+  const columnOptions = [
+    { label: 'Number of Samples', value: 'samples_count' },
+    { label: 'Samples Modality', value: 'name' },
+    { label: 'File Format', value: 'format' }
+  ]
+
+  const data = filesSummary.map((fs) =>
+    columnOptions.reduce((acc, { label, value }) => {
+      acc[label] = value === 'format' ? getReadable(fs[value]) : fs[value]
+      return acc
+    }, {})
+  )
+
+  const columns = columnOptions.map((co) => co.label)
```

Comment:
Alternatively, we can simply define  `columns` and `data` without the `columnOptions` mapper object here:

```suggestion
  const data = filesSummary.map((fs) => ({
    'Number of Samples': fs.samples_count,
    'Samples Modality': fs.name,
    'File Format': getReadable(fs.format)
  }))
  
  const columns = [ 'Number of Samples', 'Samples Modality', 'File Format' ]
```